### PR TITLE
Added some Australian News Corporation sites if you're interested :)

### PR DESCRIPTION
**Pull request by jordandalley**

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -174,6 +198,18 @@ const remove_cookies = [
...
+'couriermail.com.au',
+'heraldsun.com.au',
+'themercury.com.au',
+'ntnews.com.au',
+'geelongadvertiser.com.au',
+'cairnspost.com.au',
+'goldcoastbulletin.com.au',
+'noosanews.com.au',
+'townsvillebulletin.com.au',
+'weeklytimesnow.com.au',
```
Comment (cimke) [inline background.js]: Are you sure that all these websites has to run without cookies removed and with `google bot` functionality?

**Comment by iscreamcoke** [comment]
I'm hitting a "cookies disabled" error on all websites ("To use this website, cookies must be enabled in your browser.")

Chrome 78.0.3904.70 beta / MacOS / European Union

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ### Installation Instructions
 **Google Chrome**
-1. Download this repo as a [ZIP file from GitHub](https://github.com/iamadamdev/bypass-paywalls-chrome/archive/master.zip).
+1. Download this repo as a [ZIP file from GitHub](https://github.com/jordandalley/bypass-paywalls-chrome/archive/master.zip).
```
[the remark this answers is not in the document]
Comment (jordandalley) [reply README.md]: Yeah sorry about that. I was working on it as a fork, then decided to do a pull request :) Sorry about that

File: background.js (hunk 1 above)
Comment (jordandalley) [reply background.js]: It works for me. Not sure why it's not for you.

**Comment by jordandalley** [comment]
How strange that you're getting cookie issues. Mine have worked fine.

**Comment by iscreamcoke** [comment]
> How strange that you're getting cookie issues. Mine have worked fine.

I have reinstalled the extension from you fork (instead of replacing files) and I'm not experiencing the same issue: works like a charm now. Thanks for the PR.

**Comment by jordandalley** [comment]
Nice work :)

File: background.js [hunk 3]
```diff
@@ -107,6 +119,22 @@ const restrictions = {
...
+'asia.nikkei.com',
+'nytimes.com',
+'wsj.com',
+'ft.com',
```
Comment (iamadamdev) [inline background.js]: You are causing a lot of duplicate sites to be added to the array, only add the sites you want, I don't care about alphabetical order here. Add to the end of the array so it's easier to review the PR.

File: background.js [hunk 4]
```diff
@@ -188,6 +216,19 @@ const remove_cookies_select_drop = {
 
 // Override User-Agent with Googlebot
 const use_google_bot = [
+'theaustralian.com.au',
```
Comment (iamadamdev) [inline background.js]: don't add duplicate sites

File: background.js (hunk 3 above)
Comment (jordandalley) [reply background.js]: Sorry mate, will revise at my end. I don't use github much, so forgive the noobiness

**Comment by cimke** [comment]
@jordandalley we all are making mistakes so keep it up! We're waiting for your new PR 👍

**Comment by jordandalley** [comment]
All done mate. Hopefully the latest one passes the test :P

**Pull request closed without merging**